### ISSUE TITLE
Line indentation is inconsistent within the verse environment when \flagverse is used with text formatting

### ISSUE THREAD

**Issue by helkaluin** [bdtc/lwarp#28]
One common use case of the `\flagverse` command of the `verse` package is to print verse numbers. With LuaLaTeX, the following output is achieved in PDF (with `\vleftskip` set to 0.5em, and the `\flagverse` formatted with `\textsuperscript`):

![image](https://github.com/bdtc/lwarp/assets/6428432/5851f265-9c5f-4ea8-ac1e-6545b56b02f9)

However, when compiled with `lwarp`, the following appears (with `\HTMLvleftskip` also set to 0.5em):

![image](https://github.com/bdtc/lwarp/assets/6428432/cbdb78be-785d-455b-8361-955d15a72030)

This should not happen. The "body text" of the verse lines should all align on the left on the same vertical line. I think this might be because lwarp translates the verse environment into a `<pre>` environment in HTML, which preserves whitespace, and the HTML it generates looks like this:

![image](https://github.com/bdtc/lwarp/assets/6428432/1e530cbf-19cc-4351-b2c4-1a660f8114a3)

As you can see, the `<sup>` (and indeed things like italics and bold as well) gets ignored in the white-space counting, resulting in the inconsistent spacing. Is there a way for the HTML generated to not have this "hanging indentation"? I've included below a minimal working example.

=============================
```
\documentclass{book}
\usepackage{lwarp}
\usepackage{verse}
\setlength{\vleftskip}{0.5em}
\setlength{\HTMLvleftskip}{0.5em}

\begin{document}
  \begin{verse}
  A reading from the book of the prophet Isaiah.

  \flagverse{\textit{1}} The word that Isaiah son of Amoz saw concerning\\
  Judah and Jerusalem.

  \flagverse{\textsuperscript{2}} In days to come\\
  the mountain of the LORD's house\\
  shall be established as the highest of the mountains,\\
  and shall be raised above the hills;\\
  all the nations shall stream to it.
  \end{verse}
\end{document}
```

**Comment by bdtc** [comment]
Yes, the entire use of PRE for verse and listings and such can be a problem when things are formatted.  It may be possible to use some other system for verse especially, since PRE is usually monospace text which is not very nice for verse.  This would be something to investigate some day, but it'll be a while.  Busy at the moment.

In the meantime you could try an HTML-only definition of \flagverse without the formatting.
